bench/binary_protocol.exs

```diff
@@ -0,0 +1,56 @@
+defmodule BinaryProtocolBenchmark do
+  use Benchfella
+
+  @thrift_file_path "./test/fixtures/app/thrift/simple.thrift"
+  alias Thrift.Parser
+  alias Thrift.Protocols.Binary
+  import ParserUtils
+
+  setup_all do
+    file_group = parse_thrift(@thrift_file_path)
+    |> compile_module
+
+    {:ok, :ok}
```

Comment:
Looking at [the docs](http://elixir-lang.org/docs/stable/ex_unit/ExUnit.Callbacks.html#module-context), it appears you can return just `:ok` if you don't have any metadata to return.